docs/conf.py

```diff
@@ -174,16 +174,16 @@ def __getattr__(cls, name):
         return MagicMock()
 
 
-MOCK_MODULES = [
-    "clr",
-    "System",
-    "System.IO",
-    "NationalInstruments",
-    "NationalInstruments.VeriStand",
-    "NationalInstruments.VeriStand.ClientAPI",
-    "NationalInstruments.VeriStand.Data",
-    "NationalInstruments.VeriStand.RealTimeSequenceDefinitionApi",
-    "NationalInstruments.VeriStand.RealTimeSequenceDefinitionApiUtilities",
-    "niveristand.clientapi._datatypes.rtprimitives",
-]
-sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
+# MOCK_MODULES = [
+#     "clr",
+#     "System",
+#     "System.IO",
+#     "NationalInstruments",
+#     "NationalInstruments.VeriStand",
+#     "NationalInstruments.VeriStand.ClientAPI",
+#     "NationalInstruments.VeriStand.Data",
+#     "NationalInstruments.VeriStand.RealTimeSequenceDefinitionApi",
+#     "NationalInstruments.VeriStand.RealTimeSequenceDefinitionApiUtilities",
+#     "niveristand.clientapi._datatypes.rtprimitives",
+# ]
+# sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
```

Comment:
This throws an error with every version of Sphinx that I tried, even if I include "mock" explicitly as a module in requirements.txt, but I am also not sure it's necessary anymore, as I was able to build locally without an install of pythonnet on my machine.

Reply:
error is 
```
Exception occurred:
  File "C:\dev\git\niveristand-python\.tox\docs\lib\site-packages\sphinx\pycode\__init__.py", line 79, in for_string
    return cls(StringIO(string), modname, srcname)
TypeError: initial_value must be str or None, not MagicMock
```

Reply:
Fixed, replaced with autodoc supported mocking